.update-appimage-info.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Temporary ugly code for maintaining purposes
```

Comment:
Can you explain why this file is required and wasn't until now?

Reply:
It's not really needed, I rather added it here to make it easier for me when change the device to make changes to the current version, well, in case flathubbot does not send a pull request. This file, is not needed by anyone except me, which is why it is with a dot at the beginning.  But I can remove him..